addons/account/report/account_invoice_report.py

```diff
@@ -36,8 +36,13 @@ def _compute_amounts_in_user_currency(self, cr, uid, ids, field_names, args, con
             context={}
         currency_obj = self.pool.get('res.currency')
         currency_rate_obj = self.pool.get('res.currency.rate')
-        user_currency_id = self.pool.get('res.users').browse(cr, uid, uid, context=context).company_id.currency_id.id
-        currency_rate_id = currency_rate_obj.search(cr, uid, [('rate', '=', 1)], limit=1, context=context)[0]
+        user = self.pool.get('res.users').browse(cr, uid, uid, context=context)
+        user_currency_id = user.company_id.currency_id.id
+        currency_rate_id = currency_rate_obj.search(
+            cr, uid, [
+                ('rate', '=', 1),
+                ('currency_id.company_id', '=', user.company_id.id)
+                ], limit=1, context=context)[0]
```

Comment:
@alexis-via thanks for the fix. do you think, doing [0] on seach() might break some cases where company currency doesn't have any rate=1? I know this is unlikely usecase, but still in some case this may happen due to configuration mistake.
For instance,
- create new db
- change company currency from EUR to USD
- create a customer and invoice them
- open this customer